ols-vms.conf

```diff
@@ -0,0 +1,11 @@
+vm.architecture = amd64
+vm.release = focal
+
+[serial-vault]
+# PPAs for python-codetree and charm-tools
+apt.sources = ppa:mojo-maintainers/ppa, ppa:juju/stable
+vm.packages = @dependencies.txt, @dependencies-devel.txt, build-essential
```

Comment:
build-essential is already included in dependencies-devel.txt, no need to include it here